### ISSUE TITLE
[PGSQL] JMAP Performance tests

### ISSUE THREAD

**Issue by vttranlina** [linagora/james-project#5028]
We can wait for Postgres deployment on staging completed
Then run a performance test on the charge server -> staging
Scenario: `org.apache.james.gatling.simulation.jmap.rfc8621.PushPlatformValidationSimulation` 

Compared with JMAP Performance test on the Cassandra variant

**Comment by vttranlina** [comment]
Not sure we can compare it with Cassandra variant, 
the sample data is difference

**Comment by chibenwa** [comment]
> the sample data is difference

Yes but we would have an idea at least of the order of magnitude. 

We know we can not compare exact values for sure. But that's still interesting to run such tests.

**Comment by Arsnael** [comment]
Done in https://github.com/linagora/james-project-private/issues/784
